tsconfig.json

```diff
@@ -17,6 +17,6 @@
       "@prisma/engines-version": ["../engines-wrapper/packages/engines-version/src"]
     }
   },
-  "include": ["**/*"],
-  "exclude": ["**/dist", "**/build", "**/node_modules"]
+  "include": ["**/*", ".eslintrc.js"],
+  "exclude": ["**/dist", "**/build", "**/node_modules", "**/packages"]
```

Comment:
Excluding all packages with `"**/packages"` doesn't make sense to me, what's the logic here?

Reply:
It is necessary because you want `eslint` to only use the `tsconfig.json` of the sub-projects. Right now, it uses the base `tsconfig.json` for all sub-projects. But each project can have it's own custom inclusions and exclusions, or configuration that should be picked up per project/package.

* First, I added this config to the base `.eslintrc`
`project: ['./tsconfig.json', './packages/*/tsconfig.json'],`
Without this, `eslint` would use the base `tsconfig.json` for all packages, but that's not what we want.

* Now `eslint` applies per project, and we can exclude the sub-projects from the root (for performance).

Reply:
Great! Thanks for the explanation ❤️